src/components/Input/Input.tsx

```diff
@@ -1,8 +1,9 @@
-import _ from "lodash"
-import React, { ReactNode } from "react"
+import React, { SFC, ReactNode } from "react"
 import styled, { css } from "styled-components"
-import { BaseComponentProps, IStyled } from "../../common/types"
+import _ from "lodash"
 import { insertIf } from "../../common/_utils"
+import { IStyled } from "../../common/types"
```

Comment:
`IStyled` & `BaseComponentProps` are being imported from same file.

Reply:
feels like I should enable auto-order imports again